games-fps/freedoom/freedoom-0.11.3-r1.ebuild

```diff
@@ -5,26 +5,34 @@ EAPI=7
 
 DESCRIPTION="Free game content for the Doom engine"
 HOMEPAGE="https://freedoom.github.io"
-SRC_URI="https://github.com/freedoom/freedoom/releases/download/v${PV}/freedoom-${PV}.zip
+SRC_URI="https://github.com/freedoom/freedoom/releases/download/v${PV}/${P}.zip
 	https://github.com/freedoom/freedoom/releases/download/v${PV}/freedm-${PV}.zip"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~x86"
+KEYWORDS="~amd64 ~x86"
 IUSE=""
 
 DEPEND=""
-RDEPEND=""
+RDEPEND="
+|| (
+	games-fps/gzdoom
+	games-engines/odamex
+	games-fps/doomsday
+)
+"
 BDEPEND="app-arch/unzip"
 
 S="${WORKDIR}"
 
+FREEDOOMWADPATH=/usr/share/doom/${P}
+
 src_install() {
-	insinto /usr/share/doom-data/${PN}
+	insinto ${FREEDOOMWADPATH}
 	doins */*.wad
 	dodoc "${P}"/CREDITS.txt "${P}"/README.html
 }
 
 pkg_postinst() {
-	einfo "Please note that WAD files location is /usr/share/doom-data/${PN}"
+	einfo "Please note that the WAD files location is ${FREEDOOMWADPATH}"
```

Comment:
If this should be known to users of the package after installation, `elog` should be used instead.